package/Dockerfile

```diff
@@ -1,4 +1,7 @@
 FROM scratch
 
+ARG RKE2_DATA_DIR
+ENV RKE2_DATA_DIR=${RKE2_DATA_DIR}
```

Comment:
thinking about this more - I don't think this is necessary, is it? The image contents are extracted out to disk by rancher-system-agent and run on the host system. This Docker image is never run, so the args and env vars don't matter.